src/wiz_im.c

```diff
@@ -55,13 +55,13 @@ DEFINE_DO_FUN (do_wizhelp) {
         if (cmd_table[cmd].level >= LEVEL_HERO &&
             cmd_table[cmd].level <= char_get_trust (ch) && cmd_table[cmd].show)
         {
-            printf_to_char (ch, "%-12s", cmd_table[cmd].name);
+            printf_to_char(ch, "(%2d) %-12s", cmd_table[cmd].level, cmd_table[cmd].name);
             if (++col % 6 == 0)
```

Comment:
@rhoxie21  If we have the wiz levels levels in here, I think the number of commands per line will need to be reduced to fit an 80-column terminal. Everything else still fits in 80 columns, so I think it should be a rule.

This looks good to me:
- Reduce command space from 12 chars to 11
- Show 5 commands per row instead of 6

Reply:
Corrected!